pycalphad/io/tdb.py

```diff
@@ -764,6 +640,8 @@ def write_tdb(dbf, fd, groupby='subsystem', if_incompatible='warn'):
         output += "\n"
     # Write FUNCTION block
     for name, expr in sorted(dbf.symbols.items()):
+        if not isinstance(expr, Basic):
+            expr = sympify(expr).xreplace({Symbol('T'): v.T, Symbol('P'): v.P})
```

Comment:
Not covered - does it need a test to trigger? Should we also be replacing `R`?

Reply:
That code will probably be deleted when symengine 0.9.0 releases